docs/source/installing_in_virtualenv.rst

```diff
@@ -0,0 +1,165 @@
+Installing Nitrate on RHEL7 (virtualenv) with Apache and MySQL
```

Comment:
I prefer to remove "RHEL7" from this subject. Following content tells how to install in a virtualenv and the RHEL7 is just an example selected as the base OS. As you mentioned, these deployment steps could also apply to other distros.